go-controller/pkg/ovn/master.go

```diff
@@ -420,20 +420,6 @@ func (oc *Controller) SetupMaster(masterNodeName string, existingNodeNames []str
 		}
 	}
 
-	// Initialize the OVNJoinSwitch switch IP manager
-	// The OVNJoinSwitch will be allocated IP addresses in the range 100.64.0.0/16 or fd98::/64.
-	oc.joinSwIPManager, err = lsm.NewJoinLogicalSwitchIPManager(oc.nbClient, existingNodeNames)
```

Comment:
@JacobTanenbaum if we never use the join switch's UUID later (I looked for 5 seconds to find it so maybe I missed it) then maybe we don't need to change this part at all?

Reply:
@JacobTanenbaum fair enough.